packages/dev/eslint-config/index.js

```diff
@@ -45,7 +45,17 @@ module.exports = {
     '@parcel/no-self-package-imports': 'error',
     'import/first': 'error',
     'import/newline-after-import': 'error',
-    'import/no-extraneous-dependencies': 'error',
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        // "includeTypes": true,
+        // "packageDir": require('../../../package.json')
+        //   .workspaces
+        //   .map(workspace => require('glob').sync(workspace, {
+        //     cwd: require('path').join(__dirname, '..', '..', '..')
+        //   })),
```

Comment:
`packageDir` takes an array of packages to check. This is used to manually specify the locations of workspace packages. I do this by parsing the workspaces property on the workspace `package.json` and resolving the packages with `glob`